### DISCUSSION TITLE
Consider replacing screenshot in README.md file

### DISCUSSION THREAD

**Discussion by lucydodo** [sqlitebrowser/sqlitebrowser#3278] [General]
The screenshot for DB4S in the README.md file are outdated. (Committed Sep 8, 2013 with hash 593ef68)
I think the design of the app has changed a lot in  the meantime, what do you think about replacing them, if for no other reason? 🤔

**Comment by justinclift** [comment]
Go for it. :smile:

Make a new one that's fairly pretty, similar dimensions (width and height in pixels), and is similar in file size, then update things to use it.

**Reply by lucydodo** [reply]
![sqlitebrowser-new](https://user-images.githubusercontent.com/64626662/218895676-9641b693-5b36-4ef8-a694-1d3f5a6e08db.png)
How about this image?
The dimension is 804x639(the original image was 725x641) The DB uses Belfast Bikes Docking Stations file on the nightly server.
If you approve, I'll add the commit. 😄

**Reply by justinclift** [reply]
Heh Heh Heh.  Is it just me, or does that kind of look blurry?

That being said, it's only a *little* bit blurry.  The dimensions seem fine to me, and the file size (~140KB) isn't too big either.

If the blurriness can be improved without too much effort, then please have a go.  Otherwise, I'd just commit this new image as-is. :smile:

**Reply by lucydodo** [reply]
It looks like it got blurry during the resize process. 
I've taken a new screenshot and pushed it to my fork repo.
Go take a look (it looks a bit more blurry in the comments section 😞)

Here's the link: https://github.com/lucydodo/sqlitebrowser/tree/dev/3278

**Reply by justinclift** [reply]
Cool, that new one looks better.  Use that. :grin:

**Reply by lucydodo** [reply]
I just committed it, thanks!
